iron-form-element-behavior.html

```diff
@@ -46,8 +46,7 @@
        * The value for this element.
        */
       value: {
-        notify: true,
-        type: String
```

Comment:
I would keep the previous type, and just update the jsdoc
```
@type {string|number|null}
```

Reply:
Oh TIL, awesome. Updated :+1: